src/crimson/os/seastore/cleaner/segment_cleaner.h

```diff
@@ -368,29 +343,29 @@ class SpaceTrackerSimple : public SpaceTrackerI {
   }
 
   int64_t allocate(
-    segment_id_t segment,
-    seastore_off_t offset,
+    paddr_t addr,
     extent_len_t len) final {
-    return update_usage(segment, len);
+    return update_usage(addr.as_seg_paddr().get_segment_id(), len);
   }
 
   int64_t release(
-    segment_id_t segment,
-    seastore_off_t offset,
+    paddr_t addr,
     extent_len_t len) final {
-    return update_usage(segment, -(int64_t)len);
+    return update_usage(addr.as_seg_paddr().get_segment_id(), -(int64_t)len);
   }
 
-  int64_t get_usage(segment_id_t segment) const final {
+  int64_t get_usage(paddr_t addr) const final {
```

Comment:
I'm not sure I understand this and similar changes.  I assume the intent is to generalize it for use with RandomBlockManager, but get_usage() and other methods intended to function on segments only make sense for segmented devices.